package/MDAnalysis/core/groups.py

```diff
@@ -256,6 +257,12 @@ def setter(self, values):
             setattr(cls, attr.attrname,
                     property(getter, setter, None, attr.groupdoc))
 
+    @classmethod
+    def _whitelist(cls, attr):
+        """Allow an attribute to be set in Groups"""
+        cls._SETATTR_WHITELIST.add(attr.attrname)
+        cls._SETATTR_WHITELIST.add(attr.singular)
```

Comment:
For my understanding. Does this function update an instance of a class or the actuall class (i.e. monkey patching). The later would mean this new attr is available to all universe/atomgroup objects that are newly created.

I'm asking because of the `@classmethod` decorator.

Reply:
I need to double check this (with tests) but for each Universe we create a new version of this class.  So this is monkey patching, but we've already created a unique copy of this class for this specific Universe anyway.  So again, sort of the latter, it's available to all atomgroup objects that are created, by this Universe.